CMake/Modules/UrhoCommon.cmake

```diff
@@ -606,8 +606,14 @@ else ()
             set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${ARM_CFLAGS}")
         else ()
             if (NOT XCODE AND NOT WEB)
```

Comment:
On second thought, if my understanding is correct then I think we can revert back your proposed change below and just change the condition on this line to have a new check “AND NOT E2K” at the back. That should make thing aligned, at least to me.

Reply:
Yes, that can be done. But in any case, in code we will need a condition with checking e2k architecture (for using option -mtune=native).